lib/diagnostic.rb

```diff
@@ -28,7 +29,13 @@ def example
 # responds successfully and lists all examples.
 
 RSpec.describe 'Examples API' do
-  # your test(s) here
+  describe 'GET /examples' do
+    it 'lists all examples' do
+      get '/examples'
+
+      expect(response).to be_success
+      expect(example_response.length).to eq(example.count)
```

Comment:
Just make sure `example_response` is defined here. 

`example_response = JSON.parse(response.body)`